www/profile/app-profile.less

```diff
@@ -79,11 +84,15 @@
             margin: 5px !important;
         }
     }
+
     .cp-app-profile-resizer {
         text-align: center;
     }
+    #cp-app-profile-displayname {
+        margin-top: 1rem;
+    }
     #cp-app-profile-displayname, #cp-app-profile-link {
-        margin: 10px 0;
+        margin-bottom: 1rem;
```

Comment:
This is the expected margin bottom between buttons but other buttons are created and styled differently. You should apply the same margin-bottom to all buttons in that column. Watch out for classes that would affect all sidebar layout applications